### ISSUE TITLE
Can I use the amd backend to mine on nvidia cards !!!

### ISSUE THREAD

**Issue by adevelopcr** [fireice-uk/xmr-stak#1903]
I know it isn't as efficient as the cuda backend on nvidia cards but I wonder if ot can be used to mine on nvidia cards as well as amd cards

Cause I ask this question : I saw you talking about the cryptonight variant 2 and some people said that they tried the opencl backend on nvidia gpus but had worse results

**Comment by psychocrypt** [comment]
Yes you can use the OpenCL backend with nvidia GPUs: https://github.com/fireice-uk/xmr-stak/blob/master/doc/usage.md#nvidia-via-opencl
